src/Compilers/VisualBasic/Test/IOperation/IOperation/IOperationTests_IAnonymousObjectCreationOperation.vb

```diff
@@ -1821,5 +1821,286 @@ Block[B3] - Exit
+              Right:
+                IInvalidOperation (OperationKind.Invalid, Type: null, IsInvalid) (Syntax: '')
+                  Children(0)
+            ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: C, IsInvalid, IsImplicit) (Syntax: 'item = New C() With {.}')
+              Left:
+                ILocalReferenceOperation: item (IsDeclaration: True) (OperationKind.LocalReference, Type: C, IsImplicit) (Syntax: 'item')
+              Right:
+                IFlowCaptureReferenceOperation: 0 (OperationKind.FlowCaptureReference, Type: C, IsInvalid, IsImplicit) (Syntax: 'New C() With {.}')
+        Next (Regular) Block[B2]
+            Leaving: {R1}
+}
+Block[B2] - Exit
+    Predecessors: [B1]
+    Statements (0)
+]]>.Value
+            VerifyOperationTreeAndDiagnosticsForTest(Of MethodBlockSyntax)(source, expectedOperationTree, expectedDiagnostics)
+            VerifyFlowGraphAndDiagnosticsForTest(Of MethodBlockSyntax)(source, expectedFlowGraph, expectedDiagnostics)
+        End Sub
+
+        <Fact()>
+        Public Sub AnonymousObjectCreation_WithMissingMember_01()
+            Dim source = <![CDATA[
+Module Program
+    Sub Main() 'BIND:"Sub Main()"
+        Dim item = New With {.}
+    End Sub
+End Module
+]]>.Value
+            Dim expectedDiagnostics = <![CDATA[
+BC30201: Expression expected.
+        Dim item = New With {.}
+                              ~
+BC30203: Identifier expected.
+        Dim item = New With {.}
+                              ~
+BC30984: '=' expected (object initializer).
+        Dim item = New With {.}
+                              ~
+]]>.Value
+            Dim expectedOperationTree = <![CDATA[
+IBlockOperation (3 statements, 1 locals) (OperationKind.Block, Type: null, IsInvalid) (Syntax: 'Sub Main()  ... End Sub')
+  Locals: Local_1: item As <anonymous type: $0 As ?>
+  IVariableDeclarationGroupOperation (1 declarations) (OperationKind.VariableDeclarationGroup, Type: null, IsInvalid) (Syntax: 'Dim item = New With {.}')
+    IVariableDeclarationOperation (1 declarators) (OperationKind.VariableDeclaration, Type: null, IsInvalid) (Syntax: 'item = New With {.}')
+      Declarators:
+          IVariableDeclaratorOperation (Symbol: item As <anonymous type: $0 As ?>) (OperationKind.VariableDeclarator, Type: null) (Syntax: 'item')
+            Initializer:
+              null
+      Initializer:
+        IVariableInitializerOperation (OperationKind.VariableInitializer, Type: null, IsInvalid) (Syntax: '= New With {.}')
+          IAnonymousObjectCreationOperation (OperationKind.AnonymousObjectCreation, Type: <anonymous type: $0 As ?>, IsInvalid) (Syntax: 'New With {.}')
+            Initializers(1):
+                ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: ?, IsInvalid) (Syntax: '.')
+                  Left:
+                    IPropertyReferenceOperation: Property <anonymous type: $0 As ?>.$0 As ? (OperationKind.PropertyReference, Type: ?, IsInvalid) (Syntax: '')
```

Comment:
>Property <anonymous type: $0 As ?>.$0 As ? [](http://example.com/codeflow?start=49&length=42)

I assume this is not consistent with symbol information returned by ```SemanticModel``` for the node. Let's confirm that and assert what ```SemanticModel``` API returns for the syntax node.